programs/bubblegum/program/src/state/metaplex_adapter.rs

```diff
@@ -102,3 +102,14 @@ pub struct MetadataArgs {
     pub token_program_version: TokenProgramVersion,
     pub creators: Vec<Creator>,
 }
+
+#[derive(AnchorSerialize, AnchorDeserialize, PartialEq, Eq, Debug, Clone)]
```

Comment:
We should derive `Default` here, so we could have:
```
UpdateArgs {
  name: Some("New name"),
  ..Default::default()
}
```

Reply:
done